lighthouse-core/fraggle-rock/gather/base-artifacts.js

```diff
@@ -5,22 +5,11 @@
  */
 'use strict';
 
-const {getBrowserVersion, getBenchmarkIndex} = require('../../gather/driver/environment.js');
-
 /**
  * @param {LH.Config.FRConfig} config
- * @param {LH.Gatherer.FRTransitionalDriver} driver
  * @return {Promise<LH.BaseArtifacts>}
  */
-async function getBaseArtifacts(config, driver) {
-  const HostUserAgent = (await getBrowserVersion(driver.defaultSession)).userAgent;
-
-  // Whether Lighthouse was run on a mobile device (i.e. not on a desktop machine).
-  const HostFormFactor =
-    HostUserAgent.includes('Android') || HostUserAgent.includes('Mobile') ? 'mobile' : 'desktop';
-
-  const BenchmarkIndex = await getBenchmarkIndex(driver.executionContext);
```

Comment:
Should we keep this here so we don't need to provide a starting value for legacy base artifacts?

Reply:
might want to rebase this on top of https://github.com/GoogleChrome/lighthouse/pull/12510/files which fixes up most of our base artifact usage in FR